github-desktop: marked supported on Darwin but dependency isn't

### Nixpkgs version

- Unstable (26.05)

### Describe the bug

[Marked supported on Darwin at search.nixos.org](https://search.nixos.org/packages?channel=unstable&include_modular_service_options=1&include_nixos_options=1&query=github-desktop&show=github-desktop)

But in practice, the build fails on `aarch64-darwin`

`meta.platforms` is inherited from `pkgs.electron`: https://github.com/NixOS/nixpkgs/blob/a3854adcaf99548d2a7b3bb72710a0b7d100c63f/pkgs/by-name/gi/github-desktop/package.nix#L205

However, electron [seems to be Linux only]:https://github.com/NixOS/nixpkgs/blob/b44e340c49abc9184a9f4b4dbe65c8d4a76a3937/pkgs/development/tools/electron/common.nix#L314

### Steps to reproduce

Run `nix-build -A github-desktop --show-trace` at `nixpgs/master` and compare the result with NixOS Search info.

### Expected behaviour

a) Should be available on Darwin because the upstream app is supported
b) NixOS Search should reflect the actual scope of supported platforms to avoid confusion

### Screenshots

_No response_

### Relevant log output

```console
~/G/nixpkgs $ nix-build -A github-desktop --show-trace
error:
       … while calling the 'derivationStrict' builtin
         at <nix/derivation-internal.nix>:37:12:
           36|
           37|   strict = derivationStrict drvAttrs;
             |            ^
           38|

       … while evaluating derivation 'github-desktop-3.5.8'
         whose name attribute is located at /Users/melizare/GitHub/nixpkgs/pkgs/stdenv/generic/make-derivation.nix:541:11

       … while evaluating attribute 'buildInputs' of derivation 'github-desktop-3.5.8'
         at /Users/melizare/GitHub/nixpkgs/pkgs/stdenv/generic/make-derivation.nix:593:11:
          592|           depsHostHost = elemAt (elemAt dependencies 1) 0;
          593|           buildInputs = elemAt (elemAt dependencies 1) 1;
             |           ^
          594|           depsTargetTarget = elemAt (elemAt dependencies 2) 0;

       … in the condition of the assert statement
         at /Users/melizare/GitHub/nixpkgs/lib/customisation.nix:428:15:
          427|             outPath =
          428|               assert condition;
             |               ^
          429|               drv.${outputName}.outPath;

       … while evaluating the attribute 'handled'
         at /Users/melizare/GitHub/nixpkgs/pkgs/stdenv/generic/check-meta.nix:685:9:
          684|         valid = "no";
          685|         handled = handle {
             |         ^
          686|           inherit attrs meta;

       … from call site
         at /Users/melizare/GitHub/nixpkgs/pkgs/stdenv/generic/check-meta.nix:685:19:
          684|         valid = "no";
          685|         handled = handle {
             |                   ^
          686|           inherit attrs meta;

       … while calling 'handle'
         at /Users/melizare/GitHub/nixpkgs/pkgs/stdenv/generic/check-meta.nix:626:5:
          625|   handle =
          626|     {
             |     ^
          627|       attrs,

       … while calling the 'seq' builtin
         at /Users/melizare/GitHub/nixpkgs/pkgs/stdenv/generic/check-meta.nix:660:5:
          659|     # Give all warnings first, then error if any
          660|     builtins.seq (foldl' giveWarning null warnings) withError;
             |     ^
          661|

       … while calling the 'throw' builtin
         at /Users/melizare/GitHub/nixpkgs/pkgs/stdenv/generic/check-meta.nix:648:13:
          647|           else
          648|             throw msg;
             |             ^
          649|

       error: Refusing to evaluate package 'gnome-keyring-48.0' in /Users/melizare/GitHub/nixpkgs/pkgs/by-name/gn/gnome-keyring/package.nix:120 because it is not available on the requested hostPlatform:
         hostPlatform.system = "aarch64-darwin"
         package.meta.platforms = [
           "aarch64-linux"
           "arc-linux"
           "armv5tel-linux"
           "armv6l-linux"
           "armv7a-linux"
           "armv7l-linux"
           "i686-linux"
           "loongarch64-linux"
           "m68k-linux"
           "microblaze-linux"
           "microblazeel-linux"
           "mips-linux"
           "mips64-linux"
           "mips64el-linux"
           "mipsel-linux"
           "powerpc-linux"
           "powerpc64-linux"
           "powerpc64le-linux"
           "riscv32-linux"
           "riscv64-linux"
           "s390-linux"
           "s390x-linux"
           "x86_64-linux"
         ]
         package.meta.badPlatforms = [ ]

       a) To temporarily allow packages that are unsupported for this system, you can use an environment variable
          for a single invocation of the nix tools.

            $ export NIXPKGS_ALLOW_UNSUPPORTED_SYSTEM=1

          Note: When using `nix shell`, `nix build`, `nix develop`, etc with a flake,
                then pass `--impure` in order to allow use of environment variables.

       b) For `nixos-rebuild` you can set
         { nixpkgs.config.allowUnsupportedSystem = true; }
       in configuration.nix to override this.

       c) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
         { allowUnsupportedSystem = true; }
       to ~/.config/nixpkgs/config.nix.
```

### Additional context

_No response_

### System metadata

```
 - system: `"aarch64-darwin"`
 - host os: `Darwin 24.6.0, macOS 15.7.5`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.28.3`
 - channels(root): `"nixpkgs"`
 - nixpkgs: `/nix/store/2b2m638sjbpdsypx564aidc76vj3a3h4-source`
```

### Notify maintainers

@dtomvan 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+label%3A%220.kind%3A+bug%22+-label%3A%226.topic%3A+darwin%22+-label%3A%226.topic%3A+nixos%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

Test not yet successful :(

```log
[nix-shell:~/test]$ nix --extra-experimental-features 'nix-command flakes' build -L --keep-failed github:dtomvan/nixpkgs/6870742095fbaf5fba7a4aadf75ae92642e7f0d0#github-desktop
github-desktop> Running phase: unpackPhase
github-desktop> unpacking source archive /nix/store/mw49b84krapvpxrjzjcaa0q9b5w2qa8v-source
github-desktop> source root is source
github-desktop> Running phase: patchPhase
github-desktop> Running phase: updateAutotoolsGnuConfigScriptsPhase
github-desktop> Running phase: configurePhase
github-desktop> no configure script, doing nothing
github-desktop> Executing yarnConfigHook
github-desktop> Validating consistency between /nix/var/nix/builds/nix-5016-1916735362/source/yarn.lock and /nix/store/y72yvm3fax373ag7dwigbi2gdafkhzga-github-desktop-cache-root/yarn.lock
github-desktop> yarn config v1.21.1
github-desktop> success Set "yarn-offline-mirror" to "/nix/store/y72yvm3fax373ag7dwigbi2gdafkhzga-github-desktop-cache-root".
github-desktop> ✨  Done in 0.03s.
github-desktop> no resolved URL for package printenvz@file:./vendor/printenvz
github-desktop> Rewriting URL for local file dependency printenvz@file:./vendor/printenvz
github-desktop> yarn install v1.21.1
github-desktop> [1/4] 🔍  Resolving packages...
github-desktop> (node:5460) [DEP0169] DeprecationWarning: `url.parse()` behavior is not standardized and prone to errors that have security implications. Use the WHATWG URL API instead. CVEs are not issued for `url.parse()` vulnerabilities.
github-desktop> (Use `node --trace-deprecation ...` to show where the warning was created)
github-desktop> [2/4] 🚚  Fetching packages...
github-desktop> [3/4] 🔗  Linking dependencies...
github-desktop> warning "eslint-plugin-github > @typescript-eslint/eslint-plugin@8.21.0" has incorrect peer dependency "eslint@^8.57.0 || ^9.0.0".
github-desktop> warning "eslint-plugin-github > @typescript-eslint/eslint-plugin@8.21.0" has incorrect peer dependency "typescript@>=4.8.4 <5.8.0".
github-desktop> warning "eslint-plugin-github > @typescript-eslint/parser@8.21.0" has incorrect peer dependency "eslint@^8.57.0 || ^9.0.0".
github-desktop> warning "eslint-plugin-github > @typescript-eslint/parser@8.21.0" has incorrect peer dependency "typescript@>=4.8.4 <5.8.0".
github-desktop> warning "eslint-plugin-github > typescript-eslint@8.21.0" has incorrect peer dependency "eslint@^8.57.0 || ^9.0.0".
github-desktop> warning "eslint-plugin-github > typescript-eslint@8.21.0" has incorrect peer dependency "typescript@>=4.8.4 <5.8.0".
github-desktop> warning "eslint-plugin-github > @typescript-eslint/eslint-plugin > @typescript-eslint/type-utils@8.21.0" has incorrect peer dependency "eslint@^8.57.0 || ^9.0.0".
github-desktop> warning "eslint-plugin-github > @typescript-eslint/eslint-plugin > @typescript-eslint/type-utils@8.21.0" has incorrect peer dependency "typescript@>=4.8.4 <5.8.0".
github-desktop> warning "eslint-plugin-github > @typescript-eslint/eslint-plugin > @typescript-eslint/utils@8.21.0" has incorrect peer dependency "eslint@^8.57.0 || ^9.0.0".
github-desktop> warning "eslint-plugin-github > @typescript-eslint/eslint-plugin > @typescript-eslint/utils@8.21.0" has incorrect peer dependency "typescript@>=4.8.4 <5.8.0".
github-desktop> warning "eslint-plugin-github > @typescript-eslint/parser > @typescript-eslint/typescript-estree@8.21.0" has incorrect peer dependency "typescript@>=4.8.4 <5.8.0".
github-desktop> [4/4] 🔨  Rebuilding all packages...
github-desktop> warning Ignored scripts due to flag.
github-desktop> ✨  Done in 140.58s.
github-desktop> patching script interpreter paths in node_modules
github-desktop> node_modules/is-docker/cli.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/css-loader/node_modules/semver/bin/semver.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/jsesc/bin/jsesc: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/tldts/bin/cli.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/browserslist/cli.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/flat/cli.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/normalize-package-data/node_modules/semver/bin/semver: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/@playwright/test/cli.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/temp/node_modules/rimraf/bin.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/loose-envify/cli.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/import-local/fixtures/cli.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/he/bin/he: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/playwright-core/bin/reinstall_msedge_beta_linux.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash"
github-desktop> node_modules/playwright-core/bin/reinstall_chrome_beta_linux.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash"
github-desktop> node_modules/playwright-core/bin/reinstall_msedge_beta_mac.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash"
github-desktop> node_modules/playwright-core/bin/reinstall_msedge_stable_mac.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash"
github-desktop> node_modules/playwright-core/bin/reinstall_chrome_stable_mac.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash"
github-desktop> node_modules/playwright-core/bin/reinstall_msedge_stable_linux.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash"
github-desktop> node_modules/playwright-core/bin/reinstall_chrome_beta_mac.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash"
github-desktop> node_modules/playwright-core/bin/reinstall_msedge_dev_linux.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash"
github-desktop> node_modules/playwright-core/bin/reinstall_chrome_stable_linux.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash"
github-desktop> node_modules/playwright-core/bin/reinstall_msedge_dev_mac.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash"
github-desktop> node_modules/playwright-core/lib/utilsBundleImpl/xdg-open: interpreter directive changed from "#!/bin/sh" to "/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/sh"
github-desktop> node_modules/playwright-core/cli.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/rimraf/dist/esm/bin.mjs: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/rimraf/node_modules/glob/dist/esm/bin.mjs: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/resolve/bin/resolve: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/@eslint/eslintrc/node_modules/acorn/bin/acorn: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/global-agent/node_modules/semver/bin/semver.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/nanoid/bin/nanoid.cjs: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/acorn/bin/acorn: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/extract-zip/cli.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/prettier/bin-prettier.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/fast-xml-parser/src/cli/cli.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/tsx/dist/cli.mjs: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/typescript/bin/tsserver: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/typescript/bin/tsc: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/flat-cache/node_modules/rimraf/bin.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/@typescript-eslint/experimental-utils/node_modules/semver/bin/semver.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/@typescript-eslint/typescript-estree/node_modules/semver/bin/semver.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/@typescript-eslint/type-utils/node_modules/semver/bin/semver.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/@typescript-eslint/eslint-plugin/node_modules/semver/bin/semver.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/js-yaml/bin/js-yaml.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/ts-node/dist/bin.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/eslint-plugin-github/bin/eslint-ignore-errors.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/eslint-plugin-github/node_modules/prettier/bin/prettier.cjs: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/eslint-plugin-github/node_modules/eslint-config-prettier/bin/cli.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/cacache/node_modules/glob/dist/esm/bin.mjs: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/playwright/cli.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/eslint-plugin-jsx-a11y/node_modules/semver/bin/semver.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/damerau-levenshtein/scripts/update-changelog.sh: interpreter directive changed from "#! /usr/bin/env bash" to "/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash"
github-desktop> node_modules/esprima/bin/esparse.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/esprima/bin/esvalidate.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/legal-eagle/bin/legal-eagle: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/webpack-cli/bin/cli.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/read-installed/node_modules/semver/bin/semver: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/opener/bin/opener-bin.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/webpack/bin/webpack.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/@npmcli/fs/node_modules/semver/bin/semver.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/semver/bin/semver.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/asar/bin/asar.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/front-matter/node_modules/js-yaml/bin/js-yaml.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/electron-packager/bin/electron-packager.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/electron-packager/node_modules/semver/bin/semver.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/envinfo/dist/cli.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/ts-loader/node_modules/semver/bin/semver.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/markdown-it/bin/markdown-it.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/@babel/core/node_modules/semver/bin/semver.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/@babel/core/node_modules/json5/lib/cli.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/@babel/parser/bin/babel-parser.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/@babel/helper-compilation-targets/node_modules/semver/bin/semver.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/@babel/helpers/node_modules/@babel/parser/bin/babel-parser.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/json5/lib/cli.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/update-browserslist-db/cli.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/node-gyp/bin/node-gyp.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/node-gyp/node_modules/semver/bin/semver.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/node-gyp/node_modules/glob/dist/esm/bin.mjs: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/node-gyp/node_modules/which/bin/which.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/node-gyp/gyp/gyp_main.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/kwnbzccaiqi6iwdchcy6xc8br4x9hn0j-python3-3.13.12/bin/python3"
github-desktop> node_modules/node-gyp/gyp/pylib/gyp/common_test.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/kwnbzccaiqi6iwdchcy6xc8br4x9hn0j-python3-3.13.12/bin/python3"
github-desktop> node_modules/node-gyp/gyp/pylib/gyp/win_tool.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/kwnbzccaiqi6iwdchcy6xc8br4x9hn0j-python3-3.13.12/bin/python3"
github-desktop> node_modules/node-gyp/gyp/pylib/gyp/input_test.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/kwnbzccaiqi6iwdchcy6xc8br4x9hn0j-python3-3.13.12/bin/python3"
github-desktop> node_modules/node-gyp/gyp/pylib/gyp/__init__.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/kwnbzccaiqi6iwdchcy6xc8br4x9hn0j-python3-3.13.12/bin/python3"
github-desktop> node_modules/node-gyp/gyp/pylib/gyp/flock_tool.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/kwnbzccaiqi6iwdchcy6xc8br4x9hn0j-python3-3.13.12/bin/python3"
github-desktop> node_modules/node-gyp/gyp/pylib/gyp/generator/msvs_test.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/kwnbzccaiqi6iwdchcy6xc8br4x9hn0j-python3-3.13.12/bin/python3"
github-desktop> node_modules/node-gyp/gyp/pylib/gyp/MSVSSettings_test.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/kwnbzccaiqi6iwdchcy6xc8br4x9hn0j-python3-3.13.12/bin/python3"
github-desktop> node_modules/node-gyp/gyp/pylib/gyp/mac_tool.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/kwnbzccaiqi6iwdchcy6xc8br4x9hn0j-python3-3.13.12/bin/python3"
github-desktop> node_modules/node-gyp/gyp/pylib/gyp/easy_xml_test.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/kwnbzccaiqi6iwdchcy6xc8br4x9hn0j-python3-3.13.12/bin/python3"
github-desktop> node_modules/node-gyp/gyp/test_gyp.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/kwnbzccaiqi6iwdchcy6xc8br4x9hn0j-python3-3.13.12/bin/python3"
github-desktop> node_modules/node-gyp/gyp/gyp: interpreter directive changed from "#!/bin/sh" to "/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/sh"
github-desktop> node_modules/webpack-bundle-analyzer/lib/bin/analyzer.js: interpreter directive changed from "#! /usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/cross-env/src/bin/cross-env-shell.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/cross-env/src/bin/cross-env.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/eslint-config-prettier/bin/cli.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/espree/node_modules/acorn/bin/acorn: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/espree/node_modules/acorn-jsx/node_modules/acorn/bin/acorn: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/eslint/bin/eslint.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/eslint/node_modules/acorn/bin/acorn: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/esbuild/bin/esbuild: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/terser/bin/uglifyjs: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/terser/bin/terser: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/cssesc/bin/cssesc: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/eslint-plugin-react/node_modules/resolve/bin/resolve: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/eslint-plugin-react/node_modules/semver/bin/semver.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/electron/cli.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/sass/sass.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/html-minifier-terser/cli.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/tar/node_modules/mkdirp/bin/cmd.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/markdownlint-cli/markdownlint.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/node-test-parser/.husky/commit-msg: interpreter directive changed from "#!/usr/bin/env sh" to "/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/sh"
github-desktop> node_modules/node-test-parser/.husky/pre-commit: interpreter directive changed from "#!/usr/bin/env sh" to "/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/sh"
github-desktop> node_modules/node-test-parser/test/e2e/compare.sh: interpreter directive changed from "#!/bin/bash" to "/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash"
github-desktop> node_modules/ansi-html-community/bin/ansi-html: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/which/bin/node-which: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/node-test-github-reporter/test/e2e/compare.sh: interpreter directive changed from "#!/bin/bash" to "/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/bash"
github-desktop> node_modules/mkdirp/bin/cmd.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/@electron/get/node_modules/semver/bin/semver.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/@electron/asar/bin/asar.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/@electron/osx-sign/bin/electron-osx-flat.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/@electron/osx-sign/bin/electron-osx-sign.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/nopt/bin/nopt.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/@esbuild/android-arm/bin/esbuild: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/@esbuild/android-x64/bin/esbuild: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> finished yarnConfigHook
github-desktop> /nix/var/nix/builds/nix-5016-1916735362/source/app /nix/var/nix/builds/nix-5016-1916735362/source
github-desktop> Executing yarnConfigHook
github-desktop> Validating consistency between /nix/var/nix/builds/nix-5016-1916735362/source/app/yarn.lock and /nix/store/a3r39mxy6qk7nvlj69p0p6kpk6mpdwm2-github-desktop-cache-app/yarn.lock
github-desktop> yarn config v1.21.1
github-desktop> success Set "yarn-offline-mirror" to "/nix/store/a3r39mxy6qk7nvlj69p0p6kpk6mpdwm2-github-desktop-cache-app".
github-desktop> ✨  Done in 0.03s.
github-desktop> no resolved URL for package desktop-notifications@file:../vendor/desktop-notifications
github-desktop> Rewriting URL for local file dependency desktop-notifications@file:../vendor/desktop-notifications
github-desktop> no resolved URL for package desktop-trampoline@file:../vendor/desktop-trampoline
github-desktop> Rewriting URL for local file dependency desktop-trampoline@file:../vendor/desktop-trampoline
github-desktop> no resolved URL for package windows-argv-parser@file:../vendor/windows-argv-parser
github-desktop> Rewriting URL for local file dependency windows-argv-parser@file:../vendor/windows-argv-parser
github-desktop> yarn install v1.21.1
github-desktop> [1/4] 🔍  Resolving packages...
github-desktop> (node:6409) [DEP0169] DeprecationWarning: `url.parse()` behavior is not standardized and prone to errors that have security implications. Use the WHATWG URL API instead. CVEs are not issued for `url.parse()` vulnerabilities.
github-desktop> (Use `node --trace-deprecation ...` to show where the warning was created)
github-desktop> [2/4] 🚚  Fetching packages...
github-desktop> [3/4] 🔗  Linking dependencies...
github-desktop> [4/4] 🔨  Rebuilding all packages...
github-desktop> warning Ignored scripts due to flag.
github-desktop> ✨  Done in 37.81s.
github-desktop> patching script interpreter paths in node_modules
github-desktop> node_modules/codemirror/bin/source-highlight: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/prebuild-install/bin.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/react-transition-group/node_modules/loose-envify/cli.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/loose-envify/cli.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/node-addon-api/tools/conversion.js: interpreter directive changed from "#! /usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/detect-libc/bin/detect-libc.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/registry-js/node_modules/node-addon-api/tools/conversion.js: interpreter directive changed from "#! /usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/marked/bin/marked.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/@github/copilot/index.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/@github/copilot/npm-loader.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/fs-admin/node_modules/prebuild-install/bin.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/fs-admin/node_modules/node-addon-api/tools/conversion.js: interpreter directive changed from "#! /usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/semver/bin/semver: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/cross-spawn/node_modules/which/bin/which: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/prop-types/node_modules/loose-envify/cli.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/keytar/node_modules/prebuild-install/bin.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/keytar/node_modules/node-addon-api/tools/conversion.js: interpreter directive changed from "#! /usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/keytar/node_modules/semver/bin/semver.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/ansi-html/bin/ansi-html: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/rc/cli.js: interpreter directive changed from "#! /usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/which/bin/which.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> node_modules/mkdirp/bin/cmd.js: interpreter directive changed from "#!/usr/bin/env node" to "/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node"
github-desktop> finished yarnConfigHook
github-desktop> /nix/var/nix/builds/nix-5016-1916735362/source
github-desktop> yarn run v1.21.1
github-desktop> $ node-gyp rebuild && tsc
github-desktop> gyp info it worked if it ends with ok
github-desktop> gyp info using node-gyp@12.3.0
github-desktop> gyp info using node@24.14.1 | darwin | arm64
github-desktop> gyp info find Python using Python version 3.13.12 found at "/nix/store/kwnbzccaiqi6iwdchcy6xc8br4x9hn0j-python3-3.13.12/bin/python3"
github-desktop> gyp info spawn /nix/store/kwnbzccaiqi6iwdchcy6xc8br4x9hn0j-python3-3.13.12/bin/python3
github-desktop> gyp info spawn args [
github-desktop> gyp info spawn args '/nix/store/ddzrqa6nbfyvsrjh5grfnia4ppfkyzjl-node-gyp-12.3.0/lib/node_modules/node-gyp/gyp/gyp_main.py',
github-desktop> gyp info spawn args 'binding.gyp',
github-desktop> gyp info spawn args '-f',
github-desktop> gyp info spawn args 'make',
github-desktop> gyp info spawn args '-I',
github-desktop> gyp info spawn args '/nix/var/nix/builds/nix-5016-1916735362/source/app/node_modules/desktop-notifications/build/config.gypi',
github-desktop> gyp info spawn args '-I',
github-desktop> gyp info spawn args '/nix/store/ddzrqa6nbfyvsrjh5grfnia4ppfkyzjl-node-gyp-12.3.0/lib/node_modules/node-gyp/addon.gypi',
github-desktop> gyp info spawn args '-I',
github-desktop> gyp info spawn args '/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/include/node/common.gypi',
github-desktop> gyp info spawn args '-Dlibrary=shared_library',
github-desktop> gyp info spawn args '-Dvisibility=default',
github-desktop> gyp info spawn args '-Dnode_root_dir=/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1',
github-desktop> gyp info spawn args '-Dnode_gyp_dir=/nix/store/ddzrqa6nbfyvsrjh5grfnia4ppfkyzjl-node-gyp-12.3.0/lib/node_modules/node-gyp',
github-desktop> gyp info spawn args '-Dnode_lib_file=/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/$(Configuration)/node.lib',
github-desktop> gyp info spawn args '-Dmodule_root_dir=/nix/var/nix/builds/nix-5016-1916735362/source/app/node_modules/desktop-notifications',
github-desktop> gyp info spawn args '-Dnode_engine=v8',
github-desktop> gyp info spawn args '--depth=.',
github-desktop> gyp info spawn args '--no-parallel',
github-desktop> gyp info spawn args '--generator-output',
github-desktop> gyp info spawn args 'build',
github-desktop> gyp info spawn args '-Goutput_dir=.'
github-desktop> gyp info spawn args ]
github-desktop> gyp info spawn make
github-desktop> gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
github-desktop> make: Entering directory '/nix/var/nix/builds/nix-5016-1916735362/source/app/node_modules/desktop-notifications/build'
github-desktop>   CXX(target) Release/obj.target/desktop-notifications/src/mac/main_mac.o
github-desktop>   CXX(target) Release/obj.target/desktop-notifications/src/mac/GHDesktopNotificationsManager.o
github-desktop>   CXX(target) Release/obj.target/desktop-notifications/src/mac/Utils.o
github-desktop>   SOLINK_MODULE(target) Release/desktop-notifications.node
github-desktop> ld: warning: dylib (/nix/store/y232hw0pamh3zsmjf8ilfyq2ichnzv6q-gettext-1.0/lib/libintl.dylib) was built for newer macOS version (14.0) than being linked (11.0)
github-desktop> make: Leaving directory '/nix/var/nix/builds/nix-5016-1916735362/source/app/node_modules/desktop-notifications/build'
github-desktop> gyp info ok
github-desktop> ✨  Done in 24.41s.
github-desktop> /nix/var/nix/builds/nix-5016-1916735362/source/node_modules/printenvz /nix/var/nix/builds/nix-5016-1916735362/source
github-desktop> gyp info it worked if it ends with ok
github-desktop> gyp info using node-gyp@10.3.1
github-desktop> gyp info using node@24.14.1 | darwin | arm64
github-desktop> (node:6791) [DEP0169] DeprecationWarning: `url.parse()` behavior is not standardized and prone to errors that have security implications. Use the WHATWG URL API instead. CVEs are not issued for `url.parse()` vulnerabilities.
github-desktop> (Use `node --trace-deprecation ...` to show where the warning was created)
github-desktop> gyp info find Python using Python version 3.13.12 found at "/nix/store/kwnbzccaiqi6iwdchcy6xc8br4x9hn0j-python3-3.13.12/bin/python3"
github-desktop> gyp info spawn /nix/store/kwnbzccaiqi6iwdchcy6xc8br4x9hn0j-python3-3.13.12/bin/python3
github-desktop> gyp info spawn args [
github-desktop> gyp info spawn args '/nix/var/nix/builds/nix-5016-1916735362/source/node_modules/node-gyp/gyp/gyp_main.py',
github-desktop> gyp info spawn args 'binding.gyp',
github-desktop> gyp info spawn args '-f',
github-desktop> gyp info spawn args 'make',
github-desktop> gyp info spawn args '-I',
github-desktop> gyp info spawn args '/nix/var/nix/builds/nix-5016-1916735362/source/node_modules/printenvz/build/config.gypi',
github-desktop> gyp info spawn args '-I',
github-desktop> gyp info spawn args '/nix/var/nix/builds/nix-5016-1916735362/source/node_modules/node-gyp/addon.gypi',
github-desktop> gyp info spawn args '-I',
github-desktop> gyp info spawn args '/nix/store/qldjjimvi200yhn43dfyp5hlwl7ml169-electron-41.2.0-headers/include/node/common.gypi',
github-desktop> gyp info spawn args '-Dlibrary=shared_library',
github-desktop> gyp info spawn args '-Dvisibility=default',
github-desktop> gyp info spawn args '-Dnode_root_dir=/nix/store/qldjjimvi200yhn43dfyp5hlwl7ml169-electron-41.2.0-headers',
github-desktop> gyp info spawn args '-Dnode_gyp_dir=/nix/var/nix/builds/nix-5016-1916735362/source/node_modules/node-gyp',
github-desktop> gyp info spawn args '-Dnode_lib_file=/nix/store/qldjjimvi200yhn43dfyp5hlwl7ml169-electron-41.2.0-headers/$(Configuration)/node.lib',
github-desktop> gyp info spawn args '-Dmodule_root_dir=/nix/var/nix/builds/nix-5016-1916735362/source/node_modules/printenvz',
github-desktop> gyp info spawn args '-Dnode_engine=v8',
github-desktop> gyp info spawn args '--depth=.',
github-desktop> gyp info spawn args '--no-parallel',
github-desktop> gyp info spawn args '--generator-output',
github-desktop> gyp info spawn args 'build',
github-desktop> gyp info spawn args '-Goutput_dir=.'
github-desktop> gyp info spawn args ]
github-desktop> gyp info ok
github-desktop> /nix/var/nix/builds/nix-5016-1916735362/source
github-desktop> yarn run v1.21.1
github-desktop> $ node-gyp rebuild
github-desktop> gyp info it worked if it ends with ok
github-desktop> gyp info using node-gyp@12.3.0
github-desktop> gyp info using node@24.14.1 | darwin | arm64
github-desktop> gyp info find Python using Python version 3.13.12 found at "/nix/store/kwnbzccaiqi6iwdchcy6xc8br4x9hn0j-python3-3.13.12/bin/python3"
github-desktop> gyp info spawn /nix/store/kwnbzccaiqi6iwdchcy6xc8br4x9hn0j-python3-3.13.12/bin/python3
github-desktop> gyp info spawn args [
github-desktop> gyp info spawn args '/nix/store/ddzrqa6nbfyvsrjh5grfnia4ppfkyzjl-node-gyp-12.3.0/lib/node_modules/node-gyp/gyp/gyp_main.py',
github-desktop> gyp info spawn args 'binding.gyp',
github-desktop> gyp info spawn args '-f',
github-desktop> gyp info spawn args 'make',
github-desktop> gyp info spawn args '-I',
github-desktop> gyp info spawn args '/nix/var/nix/builds/nix-5016-1916735362/source/app/node_modules/fs-admin/build/config.gypi',
github-desktop> gyp info spawn args '-I',
github-desktop> gyp info spawn args '/nix/store/ddzrqa6nbfyvsrjh5grfnia4ppfkyzjl-node-gyp-12.3.0/lib/node_modules/node-gyp/addon.gypi',
github-desktop> gyp info spawn args '-I',
github-desktop> gyp info spawn args '/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/include/node/common.gypi',
github-desktop> gyp info spawn args '-Dlibrary=shared_library',
github-desktop> gyp info spawn args '-Dvisibility=default',
github-desktop> gyp info spawn args '-Dnode_root_dir=/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1',
github-desktop> gyp info spawn args '-Dnode_gyp_dir=/nix/store/ddzrqa6nbfyvsrjh5grfnia4ppfkyzjl-node-gyp-12.3.0/lib/node_modules/node-gyp',
github-desktop> gyp info spawn args '-Dnode_lib_file=/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/$(Configuration)/node.lib',
github-desktop> gyp info spawn args '-Dmodule_root_dir=/nix/var/nix/builds/nix-5016-1916735362/source/app/node_modules/fs-admin',
github-desktop> gyp info spawn args '-Dnode_engine=v8',
github-desktop> gyp info spawn args '--depth=.',
github-desktop> gyp info spawn args '--no-parallel',
github-desktop> gyp info spawn args '--generator-output',
github-desktop> gyp info spawn args 'build',
github-desktop> gyp info spawn args '-Goutput_dir=.'
github-desktop> gyp info spawn args ]
github-desktop> gyp info spawn make
github-desktop> gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
github-desktop> make: Entering directory '/nix/var/nix/builds/nix-5016-1916735362/source/app/node_modules/fs-admin/build'
github-desktop>   CXX(target) Release/obj.target/fs_admin/src/main.o
github-desktop> In file included from ../src/main.cc:1:
github-desktop> ../node_modules/node-addon-api/napi.h:861:39: error: in-class initializer for static data member is not a constant expression
github-desktop>   861 |     static const napi_typedarray_type unknown_array_type = static_cast<napi_typedarray_type>(-1);
github-desktop>       |                                       ^                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
github-desktop> ../node_modules/node-addon-api/napi.h:861:60: note: integer value -1 is outside the valid range of values [0, 15] for the enumeration type 'napi_typedarray_type'
github-desktop>   861 |     static const napi_typedarray_type unknown_array_type = static_cast<napi_typedarray_type>(-1);
github-desktop>       |                                                            ^
github-desktop> 1 error generated.
github-desktop> make: *** [fs_admin.target.mk:125: Release/obj.target/fs_admin/src/main.o] Error 1
github-desktop> make: Leaving directory '/nix/var/nix/builds/nix-5016-1916735362/source/app/node_modules/fs-admin/build'
github-desktop> gyp ERR! build error
github-desktop> gyp ERR! stack Error: `make` failed with exit code: 2
github-desktop> gyp ERR! stack at ChildProcess.<anonymous> (/nix/store/ddzrqa6nbfyvsrjh5grfnia4ppfkyzjl-node-gyp-12.3.0/lib/node_modules/node-gyp/lib/build.js:219:23)
github-desktop> gyp ERR! System Darwin 24.6.0
github-desktop> gyp ERR! command "/nix/store/va22g750kafjvkbwvhbnwqqmj9f9dm8m-nodejs-slim-24.14.1/bin/node" "/nix/store/ddzrqa6nbfyvsrjh5grfnia4ppfkyzjl-node-gyp-12.3.0/lib/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
github-desktop> gyp ERR! cwd /nix/var/nix/builds/nix-5016-1916735362/source/app/node_modules/fs-admin
github-desktop> gyp ERR! node -v v24.14.1
github-desktop> gyp ERR! node-gyp -v v12.3.0
github-desktop> gyp ERR! $npm_package_name fs-admin
github-desktop> gyp ERR! $npm_package_version 0.19.0
github-desktop> gyp ERR! not ok
github-desktop> error Command failed with exit code 1.
github-desktop> info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
note: keeping build directory "/nix/var/nix/builds/nix-5016-1916735362"
error: Cannot build '/nix/store/459kbci7lin9pz1vxmkanskgc80mzws1-github-desktop-3.5.8.drv'.
       Reason: builder failed with exit code 1.
       Output paths:
         /nix/store/35p9q6w4whvi5s1fhmq0hgs1qghyc2dx-github-desktop-3.5.8
       Last 25 log lines:
       > make: Entering directory '/nix/var/nix/builds/nix-5016-1916735362/source/app/node_modules/fs-admin/build'
       >   CXX(target) Release/obj.target/fs_admin/src/main.o
       > In file included from ../src/main.cc:1:
       > ../node_modules/node-addon-api/napi.h:861:39: error: in-class initializer for static data member is not a constant expression
       >   861 |     static const napi_typedarray_type unknown_array_type = static_cast<napi_typedarray_type>(-1);
       >       |                                       ^                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
       > ../node_modules/node-addon-api/napi.h:861:60: note: integer value -1 is outside the valid range of values [0, 15] for the enumeration type 'napi_typedarray_type'
       >   861 |     static const napi_typedarray_type unknown_array_type = static_cast<napi_typedarray_type>(-1);
       >       |                                                            ^
       > 1 error generated.
       > make: *** [fs_admin.target.mk:125: Release/obj.target/fs_admin/src/main.o] Error 1
       > make: Leaving directory '/nix/var/nix/builds/nix-5016-1916735362/source/app/node_modules/fs-admin/build'
       > gyp ERR! build error
       > gyp ERR! stack Error: `make` failed with exit code: 2
       > gyp ERR! stack at ChildProcess.<anonymous> (/nix/store/ddzrqa6nbfyvsrjh5grfnia4ppfkyzjl-node-gyp-12.3.0/lib/node_modules/node-gyp/lib/build.js:219:23)
       > gyp ERR! System Darwin 24.6.0
       > gyp ERR! command "/nix/store/va22g750kafjvkbwvhbnwqqmj9f9dm8m-nodejs-slim-24.14.1/bin/node" "/nix/store/ddzrqa6nbfyvsrjh5grfnia4ppfkyzjl-node-gyp-12.3.0/lib/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
       > gyp ERR! cwd /nix/var/nix/builds/nix-5016-1916735362/source/app/node_modules/fs-admin
       > gyp ERR! node -v v24.14.1
       > gyp ERR! node-gyp -v v12.3.0
       > gyp ERR! $npm_package_name fs-admin
       > gyp ERR! $npm_package_version 0.19.0
       > gyp ERR! not ok
       > error Command failed with exit code 1.
       > info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
       For full logs, run:
         nix log /nix/store/459kbci7lin9pz1vxmkanskgc80mzws1-github-desktop-3.5.8.drv
```

--- a/pkgs/by-name/gi/github-desktop/package.nix
+++ b/pkgs/by-name/gi/github-desktop/package.nix
@@ -202,6 +202,6 @@ stdenv.mkDerivation (finalAttrs: {
     license = lib.licenses.mit;
     mainProgram = "github-desktop";
     maintainers = with lib.maintainers; [ dtomvan ];
-    inherit (electron.meta) platforms;
+    platforms = lib.lists.intersectLists electron.meta.platforms lib.platforms.linux;
   };
 })

